Form-Controls/index.html

```diff
@@ -9,19 +9,51 @@
   </head>
   <body>
     <header>
-      <h1>Product Pick</h1>
+      <h1>Product Pick</h1>  
     </header>
     <main>
-      <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
-      </form>
+<form>
```

Comment:
The formatting is slightly off here - each new element should be indented

You can do this within VS code by using `Format document` - https://code.visualstudio.com/docs/editing/codebasics#_formatting